From adcc9739e6457df20339161299cf267de9699a2e Mon Sep 17 00:00:00 2001
From: Sam Shih <sam.shih@mediatek.com>
Date: Fri, 2 Jun 2023 13:06:20 +0800
Subject: [PATCH] 
 [spi-and-storage][999-2370-Modify-tick_delay-for-spi-work-safety.patch]

---
 drivers/spi/spi-mt65xx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-mt65xx.c b/drivers/spi/spi-mt65xx.c
index 7c3ee3381..83a2417a8 100644
--- a/drivers/spi/spi-mt65xx.c
+++ b/drivers/spi/spi-mt65xx.c
@@ -195,7 +195,7 @@ static const struct mtk_spi_compatible mt8183_compat = {
  */
 static const struct mtk_chip_config mtk_default_chip_info = {
 	.sample_sel = 0,
-	.get_tick_dly = 1,
+	.get_tick_dly = 2,
 };
 
 static const struct of_device_id mtk_spi_of_match[] = {
-- 
2.34.1

